core/connection.go

```diff
@@ -1,17 +1,28 @@
 package core
 
 import (
+	"sync/atomic"
+
 	"github.com/yomorun/yomo/core/frame"
 	"github.com/yomorun/yomo/core/metadata"
 	"golang.org/x/exp/slog"
 )
 
+var increment uint64
+
+// incrID generates next increment ID.
+func incrID() uint64 {
+	return atomic.AddUint64(&increment, 1)
+}
+
 // ConnectionInfo holds the information of connection.
 type ConnectionInfo interface {
+	// ID is the ID generated by server.
+	ID() uint64
+	// ClientID represents a client ID, the ClientID generated by client.
+	ClientID() string
```

Comment:
Can the client ID be deleted?

Reply:
I think we can keep it for now. We can trace the client by the clientID. 

And the clientID is from handshakeFrame.